src/components/MainContent/MainContent.js

```diff
@@ -1,13 +1,14 @@
 import React, { Component } from 'react';
+import * as playersActions from '../../modules/players/actions';
```

Comment:
what is this?

Reply:
That was written by Andres, so I don't know, but I question him and  I resolve the problem.